.cargo/config

```diff
@@ -1,2 +1,5 @@
+# Copyright (C) 2022 Nicolas Stalder
```

Comment:
I'm not a fan at all of adding these headers to EVERY SINGLE FILE.

Can we not simple in the README detail that docs/examples/scripts are CC0, and source code is Apache2 OR MIT, with contributions from our two companies, and whoever may contribute (or already has contributed...)? Personally I'd even prefer just something like "The Trussed developers" as "author".

If we have concerns about copyright in the future, we could start requiring contributions have the "sign-off", the way Linux does it to signal explicit agreement with the standard clause:

<sup>`piv-authenticator` is licensed under either of [Apache License, Version 2.0](https://github.com/trussed-dev/trussed/blob/main/LICENSE-APACHE) or [MIT License](https://github.com/trussed-dev/trussed/blob/main/LICENSE-MIT) at your option.</sup>
<sub>Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any additional terms or conditions.</sub>

Reply:
In the specific case of Linux that you mention they do have an `// SPDX-License-Identifier: GPL-2.0` header in every file. This also allows us to use `reuse` to check the licencing.

They don't have the `Copyright (C) <year>` line though. We can probably remove it.

I'm in favor of adding the following to the readme: 

> Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in `piv-authenticator` by you, shall be licensed as MIT OR Apache-2.0, without any additional terms or conditions.

Reply:
I removed the headers.